src/App.jsx

```diff
@@ -14,70 +16,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function isSelected(good) {
+    return selectedGood === good;
+  }
```

Comment:
don't create a function for this check, use just a const

and its better to give more clear name ...what exactly is selected?

```
 <tbody>
  {goods.map((good) => {
    const isGoodSelected = selectedGood === good;
```